bin/run_tests

```diff
@@ -13,6 +13,12 @@ cd core
 nose2 -v
 cd ..
 
+PYTHONPATH=$top_dir/signing
+export PYTHONPATH
```

Comment:
Think you might be missing this before exporting pythonpath
PYTHONPATH=$PYTHONPATH:$top_dir/signing/build/lib.linux-x86_64-2.7